src/index_runner/indexers/main.py

```diff
@@ -111,7 +114,8 @@ def _default_fields(obj_data, ws_info, obj_data_v1):
     {'module': 'KBaseNarrative', 'type': 'Narrative', 'indexer': index_narrative},
     {'module': 'KBaseFile', 'type': 'PairedEndLibrary', 'indexer': index_reads},
     {'module': 'KBaseFile', 'type': 'SingleEndLibrary', 'indexer': index_reads},
-    {'module': 'KBaseGenomeAnnotations', 'type': 'Assembly', 'indexer': index_assembly}
+    {'module': 'KBaseGenomeAnnotations', 'type': 'Assembly', 'indexer': index_assembly},
+    {'module': 'KBaseGenomes', 'type': 'Genome', 'indexer': index_genome}
```

Comment:
Do you want to constrain the type here? I think you can add "version" in this dictionary. You'd have to repeat the same indexers for all the versions you want. We could also just let it fail for incompat versions

Reply:
I've wrapped most everything with `.get()` so older version  will fail silently. Can't decide if thats a good thing or not

Reply:
i think we should probably have a larger discussion about how we want to handle different versions

Reply:
What I was thinking:
- All type versions have the same alias (eg. "genome"). We may have "genome-15:1", "genome-16:1", etc, with all different indexers, and they are all aliased to "genome". So when you search "genome", you are searching all versions.
- Instead of an exact match on version inside `_find_indexer`, we could have `min_version` and `max_version` fields so we don't have to repeat it so much.